### PR TITLE
Wiki Page

### PR DESCRIPTION
**Pull request by MasterBugPatch**
Fixed container changes, navbar disappears on low widths (e.g.: mobile).
Complete wiki page with route change towards it.

### CODE REVIEW

**Comment by MasterBugPatch** [comment]
@peppy Finally got to the fixes left off from https://github.com/ppy/osu-web/pull/298

File: resources/assets/less/wiki.less [hunk 1]
```diff
@@ -16,23 +17,178 @@
...
+    margin-top: 10px;
+    margin-bottom: 10px;
+}
 
-    h1, h2 {
-      .default-text-shadow;
-      color: #fff;
+.wiki-nav-container {
+    right: 30px;
+    .wiki-button {
```
Comment (nanaya) [inline resources/assets/less/wiki.less]: newer styles should only be written in bem https://css-tricks.com/bem-101/

File: resources/assets/less/wiki.less [hunk 2]
```diff
@@ -16,23 +17,178 @@
...
+        span {
+            font-size: 170%;
```
Comment (nanaya) [inline resources/assets/less/wiki.less]: why is the font size in percentage

File: resources/assets/less/wiki.less [hunk 3]
```diff
@@ -16,23 +17,178 @@
...
+            color: #0088AB;
+        }
+    }
+}
 
-      font-style: normal;
+.wiki-nav-bottom {
+    //container
+    &:extend(.row);
+    z-index: 1000;
```
Comment (nanaya) [inline resources/assets/less/wiki.less]: all z-index numbers must be declared in `variables.less`

File: resources/assets/less/wiki.less [hunk 4]
```diff
@@ -16,23 +17,178 @@
...
+    position: fixed;
+    width: 15%;
+    max-width: 160px;
+    text-align: center;
+    bottom: 25%;
```
Comment (nanaya) [inline resources/assets/less/wiki.less]: this looks like randomly positioned

File: resources/assets/less/wiki.less [hunk 5]
```diff
@@ -16,23 +17,178 @@
...
+        }
+    }
+    .top {
+        margin-top: -14%;
+        .wiki-inner {
+            font-size: 60%;
+            color: #FFF;
+        }
+        .wiki-outer {
+            color: #0088AB;
```
Comment (nanaya) [inline resources/assets/less/wiki.less]: use available one of available color variables

File: resources/assets/less/wiki.less [hunk 6]
```diff
@@ -16,23 +17,178 @@
...
+
+#hideWikiNav:hover {
+    opacity: 0.75;
+}
+
+#showWikiNav:hover {
+    opacity: 0.75;
 }
+
+.wiki-page {
```
Comment (nanaya) [inline resources/assets/less/wiki.less]: this top class isn't even needed. Just use the page-compact modifier of osu-layout__row.

File: resources/views/help/wiki.blade.php [hunk 7]
```diff
@@ -16,3 +17,253 @@
...
+    $('#hideWikiNav').click(function() {
+        $("#wikinav").hide();
+        $("#showWikiNav").show();
+    });
+	
+	$(window).resize(function() {
+		checkNavSize();
+	});
+	
+	function checkNavSize(){
```
Comment (nanaya) [inline resources/views/help/wiki.blade.php]: learn about media query https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries

File: resources/views/help/wiki.blade.php [hunk 8]
```diff
@@ -16,3 +17,253 @@
...
+                <a href="">osu! Alumni</a> •
+                <a href="">Community Contributors</a> •
+                <a href="">List of notable people</a> •
+                <a href="">Support Team</a>
+            </div>
+        </div>
+    </div>
+</div>
+
+<script>
```
Comment (nanaya) [inline resources/views/help/wiki.blade.php]: no inlining of javascript.

File: resources/views/help/wiki.blade.php [hunk 9]
```diff
@@ -16,3 +17,253 @@
...
+
+            <div class="wiki-button top">
+                <span id="hideWikiNav" class="fa-stack">
+			<div class="wiki-outer"><i class="fa fa-circle fa-stack-2x"></i></div>
+			<div class="wiki-inner"><i class="fa fa-bars fa-stack-1x"></i></div>
+		</span>
+            </div>
+
+            <br>
+            <b><i>Wiki Navigation</i></b>
```
Comment (nanaya) [inline resources/views/help/wiki.blade.php]: what's with all this `<b><i>`. Try using css.

File: resources/views/help/wiki.blade.php [hunk 10]
```diff
@@ -16,3 +17,253 @@
...
+</div>
+
+<div class='osu-layout__row osu-layout__row--page wiki-page'>
+
+    <div class="wiki-nav-container">
+
+        <div id="wikinav" class="wiki-nav">
+
+            <div class="wiki-button top">
+                <span id="hideWikiNav" class="fa-stack">
```
Comment (nanaya) [inline resources/views/help/wiki.blade.php]: javascript related identifiers must use class, prefixed with `js-`

File: resources/views/help/wiki.blade.php [hunk 11]
```diff
@@ -16,3 +17,253 @@
     along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
 --}}
 @extends("master")
+
+@section("content")
+<div class="osu-layout__row osu-layout__row--page-compact">
+    <div class="osu-page-header osu-page-header--wiki">
+        <h1 class="osu-page-header__title"><i>Getting Started</i></h1>
```
Comment (nanaya) [inline resources/views/help/wiki.blade.php]: what's `<i>` doing in here

File: resources/assets/less/wiki.less [hunk 12]
```diff
@@ -16,23 +17,178 @@
...
+    background-color: #FFFFFF;
+    a {
+        display: block;
+        margin-bottom: 3px;
+        text-align: center;
+    }
+}
+
+#hideWikiNav:hover {
+    opacity: 0.75;
```
Comment (nanaya) [inline resources/assets/less/wiki.less]: I don't remember other part of site using this kind of hover effect.

**Comment by MasterBugPatch** [comment]
@nanaya Alright, I'll try to make these changes tonight. Thanks for taking the time to correct it

**Pull request closed without merging**